docs/configuration/misc-options.md

```diff
@@ -55,6 +55,8 @@ To allow time for players to finish what they're doing during a graceful server
     
     The grace period can be increased using [the -t option on docker-compose down](https://docs.docker.com/compose/reference/down/) or set the [stop_grace_period](https://docs.docker.com/compose/compose-file/05-services/#stop_grace_period) in the compose file.
 
+The `STOP_SERVER_ANNOUNCE_DELAY` can be bypassed by sending a `SIGUSR1` signal to the `mc-server-runner` process.

```

Comment:
Over here it would be better to describe in terms of the `-s` or `--signal` option for docker stop and docker compose stop.

Reply:
Good point, will add it :)

Reply:
I've added examples for `docker` and `docker compose`. :)